providers/implementations/keymgmt/ec_kmgmt.c

```diff
@@ -440,6 +441,8 @@ int common_import(void *keydata, int selection, const OSSL_PARAM params[],
 
         ok = bnctx != NULL && ossl_ec_key_pairwise_check(ec, bnctx);
         BN_CTX_free(bnctx);
+        if (ok <= 0)
+            ossl_set_error_state(OSSL_SELF_TEST_TYPE_PCT);
```

Comment:
Do we enter error state on any failure or only on the failure of the PCT?

Reply:
Failing to run the PCT would count as a critical error too.  At least I think it would.

That means the BN CTX allocation failure will induce the error state.